libraries/AP_Scripting/docs/docs.lua

```diff
@@ -4002,6 +4002,19 @@ function fence:get_margin_breaches() end
 ---@return number -- distance
 function fence:get_breach_distance(fence_type) end
 
+-- Rally library
+rally = {}
+
+-- Returns the count of the number of Rally points
+---@return integer
+function rally:get_rally_total() end
+
+-- Returns a specfic rally by index as a Location (i.e NOT as a RallyLocation, a regular Location)
+---@param index integer
+---@param rally_location Location_ud
+---@return boolean
+function rally:get_rally_location_with_index(index, rally_location) end
```

Comment:
```suggestion
---@return rally_location Location_ud|nil
function rally:get_rally_location_with_index(index) end
```
`Location'Null` in the bindings means the location is returned if the function returns true, else nil is returned.

Reply:
Thanks @IamPete1  - Almost! the return can't have a name so its

`---@return Location_ud|nil`

Which works